domains/jam.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jamzhou",
+        "email": "jie87911@126.com"
+    },
+    "record": {
+        "CNAME": "jamzhou.github.io/"
```

website doesnt contain anything
CNAME can't contain slashes
```suggestion
        "CNAME": "jamzhou.github.io"
```